src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -138,25 +145,9 @@ private AIState prepareForFarming()
 
         if (currentField.needsWork())
         {
-            switch (currentField.getFieldStage())
-            {
-                case EMPTY:
-                    if (!checkForHoe())
-                    {
-                        return walkToBlock(currentField.getLocation()) ? AIState.PREPARING : AIState.FARMER_HOE;
-                    }
-                    break;
-                case HOED:
-                    if (canGoPlanting(currentField, building))
-                    {
-                        return walkToBlock(currentField.getLocation()) ? AIState.PREPARING : AIState.FARMER_PLANT;
-                    }
-                    break;
-                case PLANTED:
-                    return walkToBlock(currentField.getLocation()) ? AIState.PREPARING : AIState.FARMER_HARVEST;
-                default:
-                    break;
-            }
+        	if(!checkForHoe()&& canGoPlanting(currentField, building)){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)